docs/whats-new.rst

```diff
@@ -28,6 +28,9 @@ Corrected link to documentation badge on repository main page (:pull:213). By Da
 Enhancements
 ~~~~~~~~~~~~
 
+- Create Travis branch that tests against the xarray dev branch. This
```

Comment:
"Create Travis CI environment that tests against the xarray development branch."  Omit second sentence.  Move from Enhancements to new section "Testing"